src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -60,6 +60,35 @@ public String toString() {
     }
   };
 
+  public static final Builder<Class<?>> CLASS = new Builder<Class<?>>() {
```

Comment:
1. rename it more descriptive and unique; i.e. REDIS_JSON_TYPE
2. move it after all core Redis related builders; i.e. after STR_ALGO_LCS_RESULT